test/utils/compute/compute.go

```diff
@@ -0,0 +1,101 @@
+package compute
```

Comment:
move to test/utils/azure/compute/compute.go?

add package comment,

https://go.dev/wiki/CodeReviewComments#doc-comments
`All top-level, exported names should have doc comments, as should non-trivial unexported type or function declarations. `